staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -42,10 +42,13 @@ import (
 type AdmissionOptions struct {
 	// RecommendedPluginOrder holds an ordered list of plugin names we recommend to use by default
 	RecommendedPluginOrder []string
-	// DefaultOffPlugins a list of plugin names that should be disabled by default
-	DefaultOffPlugins []string
-	PluginNames       []string
-	ConfigFile        string
+	// DefaultEnabledPluginSet is a set of plugin names that is enabled by default
+	DefaultEnabledPluginSet sets.String
+	// DEPRECATED FLAGS
```

Comment:
which of these are deprecated? Just PluginNames, right? Clarify that in the comment

Reply:
Also, should the old flag and the new flags be mutually exclusive? I don't think allowing both makes sense. Or, should we expand the existing flag to allow `--admission-plugin=+foo,-bar` format?

Reply:
Yes, If both flags are specified, only the old flag is valid.